corehq/apps/data_interfaces/static/data_interfaces/js/case_property_input.js

```diff
@@ -56,10 +56,12 @@ hqDefine('data_interfaces/js/case_property_input', [
             self.placeholder = gettext("case property name");
         },
         template: '<div>\
-          <select class="form-control"\
-                  required\
-                  data-bind="visible: showDropdown, value: valueObservable, autocompleteSelect2: casePropertyNames"\
-          ></select>\
+          <!-- ko if: showDropdown -->\
```

Comment:
Since this doesn't vary per pageload, another option would be to exclude it from this template entirely when `!showDropdown`, though i guess that's about the same.

Reply:
I don't know that `self.showDropdown` would be accessible from here (although now I'm curious).